src/components/AllBooks/allBooks.styles.js

```diff
@@ -3,14 +3,17 @@ import styled from "styled-components";
 export const Container = styled.div`
   display: flex;
   flex-wrap: wrap;
+  justify-content: center;
+  max-width: 2000px;
```

Comment:
use something more dynamic like max-width: 100% or 90% or vh based on how much space you want it to take, it will help a lot for responsive design, I also prefer "rem" instead of pixels which is based on the "root em" which will return you a percentage based scaling value.